✨Feat: 특정 게임 조회 / 🔨Refactor: 게임 전체 조회

Rosaly-J opened the pull request:
✨Feat: 특정 게임 조회
- swagger 추가
- 특정 게임 조회 api 기능 구현
- 테스트 통과

🔨Refactor: 게임 전체 조회
- 게임 조회에서 전체 조회로 변경
- sort 기능 삭제

Rosaly-J committed:
✨Feat: 특정 게임 조회

- swagger 추가
- 특정 게임 조회 api 기능 구현
- 테스트 통과

🔨Refactor: 게임 전체 조회
- 게임 조회에서 전체 조회로 변경
- sort 기능 삭제

hwangtate commented on src/games/views/game_view.py:
```diff
@@ -27,34 +34,71 @@ class GameListView(APIView):
+            limit = request.query_params.get("limit")
+            games = Game.objects.all()
+
+            if limit is not None:
```
쿼리 파라미터로 limit을 받는 부분 삭제 부탁드립니다!

Rosaly-J committed:
✨Feat: 특정 게임 조회

- swagger 추가
- 특정 게임 조회 api 기능 구현
- 테스트 통과

🔨Refactor: 게임 전체 조회

- 게임 조회에서 전체 조회로 변경
- sort 기능 삭제
